admin/test/test_acceptance.py

```diff
@@ -205,3 +208,27 @@ def test_non_json_message(self):
             )],
             self._convert(NON_JSON_JOURNAL_EXPORT),
         )
+
+
+class CommonOptionsTests(TestCase):
+    """
+    Tests for ``CommonOptions``.
+    """
+    def test_invalid_version(self):
+        """
+        An ``UsageError`` is thrown if an invalid flocker version is passed
+        to the ``CommonOptions``.
+        """
+        arg_options = (
+            "--distribution ubuntu-14.04 --provider AWS "
+            "--flocker-version invalid-version"
+        )
```

Comment:
I might also make `invalid-version` a variable and then make an assertion that the invalid option_value is included in the error message...which is a nice touch by the way. There's nothing more annoying than a validation error that doesn't include the invalid value.